iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -918,15 +918,15 @@ private static void testDesiredPropertiesFlow(DeviceClient deviceClient, TwinCli
     private static void testReportedPropertiesFlow(DeviceClient deviceClient, TwinClient twinClientServiceClient, String expectedPropertyKey, String expectedPropertyValue) throws IOException, IotHubException, InterruptedException, TimeoutException, IotHubClientException
     {
         String expectedReportedPropertyValue = expectedPropertyValue + "-reported";
-        TwinCollection reportedProperties = new TwinCollection();
+        TwinCollection reportedProperties = deviceClient.getTwin().getReportedProperties();
         reportedProperties.put(expectedPropertyKey, expectedReportedPropertyValue);
         deviceClient.updateReportedProperties(reportedProperties);
 
         // Verify that the new reported property value can be seen from the service client
         Twin serviceClientTwin = twinClientServiceClient.get(deviceClient.getConfig().getDeviceId());
 
         com.microsoft.azure.sdk.iot.service.twin.TwinCollection retrievedReportedProperties = serviceClientTwin.getReportedProperties();
-        assertEquals(1, retrievedReportedProperties.size());
+        assertTrue(retrievedReportedProperties.size() > 0);
```

Comment:
Twin may have more than one property if the identity under test has been used prior to this test